Ginger/Ginger/SolutionWindows/TreeViewItems/DataSourceTreeItem.cs

```diff
@@ -180,7 +184,14 @@ private void AddNewCustomizedTable(object sender, System.Windows.RoutedEventArgs
                 inValidTableNameChars[pathChars.Length] = ' ';
                 if (GingerCore.GeneralLib.InputBoxWindow.GetInputWithValidation("Add New Customized Table", "Table Name", ref name, inValidTableNameChars))
                 {
-                    CreateTable(name, "[GINGER_ID] AUTOINCREMENT,[GINGER_USED] Text,[GINGER_LAST_UPDATED_BY] Text,[GINGER_LAST_UPDATE_DATETIME] Text", DataSourceTable.eDSTableType.Customized);
+                    if (DSDetails.DSType == DataSourceBase.eDSType.MSAccess)
```

Comment:
too many if/else and assumption we have only access and LiteDB - we will convert to OO style